dom.bs

```diff
@@ -3321,9 +3322,10 @@ method, when invoked, must run these steps:
   <a for="registered observer">observer</a> is the <a>context object</a>:
 
   <ol>
-   <li><p>Remove all <a>transient registered observers</a> whose <b>source</b> is
-   <var>registered</var>.
-   <!-- XXX remove from where? -->
+   <li><p><a for=list>For each</a> <var>node</var> of the <a>context object</a>'s
+   <a for=MutationObserver>node list</a>, <a for=list>remove</a> all
+   <a>transient registered observers</a> whose <a for="transient registered observers">source</a> is
+   <var>registered</var> from <var>node</var>'s <a>registered observer list</a>.
```

Comment:
Something weird with this sentence.

Reply:
Can you be more specific? I can't really find anything odd.